cookbook-react-native/recipes/buttons/custom-radio-button-group/components/RadioButton/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { Key } from 'react';
+import { TouchableOpacity, ViewStyle, View, TextProps } from 'react-native';
+import CustomText from '@textsRecipes/custom-text';
+
+import styles from './styles';
+import { getBackgroundColor, getBorderColor } from './utils';
+
+interface Props<K, T> {
+  option: T;
+  optionKey: K;
+  selected: boolean;
+  disabled?: boolean;
+  onPress: (option: K) => void;
```

Comment:
since you above said that `optionKey`'s type is `K`, i would change to
```suggestion
  onPress: (optionKey: K) => void;
```